<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PipFreezeDialog</class>
 <widget class="QDialog" name="PipFreezeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Requirements</string>
  </property>
  <property name="toolTip">
   <string>Replace the current selection with the requirements text</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="localCheckBox">
     <property name="toolTip">
      <string>Select to show requirements for locally-installed packages only</string>
     </property>
     <property name="text">
      <string>Local packages only</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Requirements File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="E5PathPicker" name="requirementsFilePicker" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="saveButton">
       <property name="toolTip">
        <string>Press to save to the requirements file</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="saveToButton">
       <property name="toolTip">
        <string>Save to a new file</string>
       </property>
       <property name="text">
        <string>Save To</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="copyButton">
       <property name="toolTip">
        <string>Copy the requirements text to the clipboard</string>
       </property>
       <property name="text">
        <string>Copy</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="insertButton">
       <property name="toolTip">
        <string>Insert the requirements text at the cursor position</string>
       </property>
       <property name="text">
        <string>Insert</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="replaceSelectionButton">
       <property name="text">
        <string>Replace Selection</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="replaceAllButton">
       <property name="toolTip">
        <string>Replace all text with the requirements text</string>
       </property>
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" rowspan="7">
      <widget class="QPlainTextEdit" name="requirementsEdit">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>E5PathPicker</class>
   <extends>QWidget</extends>
   <header>E5Gui/E5PathPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>localCheckBox</tabstop>
  <tabstop>requirementsFilePicker</tabstop>
  <tabstop>requirementsEdit</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>saveToButton</tabstop>
  <tabstop>copyButton</tabstop>
  <tabstop>insertButton</tabstop>
  <tabstop>replaceSelectionButton</tabstop>
  <tabstop>replaceAllButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PipFreezeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PipFreezeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
